src/main/resources/reference.conf

```diff
@@ -0,0 +1,16 @@
+das {
+    server {
+        port = 50051 # the port the server listens on
+        monitoring-port = 8080 # http port for monitoring
+        max-chunk-size = 1000 # the maximum number of rows that can be returned in a single chunk
```

Comment:
What is this one, and why different from batch-size below?
Is it necessary or can be done 'time-based' chunking only?

We have to be careful to not "over-fetch" from the source systems.